doc/doxygen/src/changelog-deprecated.md

```diff
@@ -0,0 +1,6 @@
+# Changelog (Deprecated)                                       {#changelog}
+
+@deprecated This page is deprecated.
+See [Changelog](https://guide.riot-os.org/misc/changelog)
+on the RIOT Guide Site for the latest information.
+This page will be removed after release 2026.01.
```

Comment:
```suggestion
This page will be removed after release 2026.04.
```

If you want to conform with the other deprecations that should be 2026.04, (2 releases in the future)

Reply:
Fixed